std_msgs/Header header

string[] names
float32[] probabilities
